### PR TITLE
Add handle by user api for classified exception

### PR DESCRIPTION
**Pull request by lcadzy**

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/common/classifiedexception/ClassifiedException.kt [hunk 1]
```diff
@@ -36,4 +36,7 @@ abstract class ClassifiedException(exp: Throwable?) : Throwable(exp), ILogger {
     fun logStackTrace() {
         log().warn("$title: $stackTrace")
     }
+
+    open fun handleByUser(){
```
Comment (wezhang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/common/classifiedexception/ClassifiedException.kt]: `abstract`?

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/common/classifiedexception/SparkToolException.kt [hunk 2]
```diff
@@ -21,13 +21,23 @@
...
 import org.apache.commons.lang.exception.ExceptionUtils
 
 const val ToolPackageSuffix: String = "com.microsoft.azure"
 
 class SparkToolException(exp: Throwable?) : ClassifiedException(exp) {
     override val title: String = "Azure Plugin for IntelliJ Error"
+
+    override fun handleByUser(){
+        ApplicationManager.getApplication().invokeLater {
+            val em = ErrorMessageForm(title)
```
Comment (wezhang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/common/classifiedexception/SparkToolException.kt]: `toolErrorDialog`

**Comment by Eskibear** [comment]
retest this please

**Comment by wezhang** [comment]
retest this please

**Comment by Eskibear** [comment]
retest this please

**Pull request merged**